example/bin/monorail_rack

```diff
@@ -61,16 +54,20 @@ if [ $pxe_count ]
   then
     for (( i=1; i <= $pxe_count; i++ ))
       do
-        echo "deploying pxe: $i"
         vmName="pxe-$i"
-
-        vboxmanage createvm --name $vmName --register;
-        vboxmanage createhd --filename $vmName --size 8192;
-        VBoxManage storagectl $vmName --name "SATA Controller" --add sata --controller IntelAHCI
-        VBoxManage storageattach $vmName --storagectl "SATA Controller" --port 0 --device 0 --type hdd --medium $vmName.vdi
-        vboxmanage modifyvm $vmName --ostype Ubuntu --boot1 net --memory 350;
-        vboxmanage modifyvm $vmName --nic1 intnet --intnet1 closednet --nicpromisc1 allow-all;
-        vboxmanage modifyvm $vmName --nictype1 82540EM --macaddress1 auto;
-
+        if [[ ! -e $vmName.vdi ]]; then # check to see if PXE vm already exists
+            echo "deploying pxe: $i"
+            vboxmanage createvm --name $vmName --register;
+            vboxmanage createhd --filename $vmName --size 8192;
+            vboxmanage storagectl $vmName --name "SATA Controller" --add sata --controller IntelAHCI
+            vboxmanage storageattach $vmName --storagectl "SATA Controller" --port 0 --device 0 --type hdd --medium $vmName.vdi
+            vboxmanage modifyvm $vmName --ostype Ubuntu --boot1 net --memory 768;
```

Comment:
I tried with both, and was able to get away with 768MB, so I thought I'd try and keep it as small as possible from a quick/dev setup perspective